src/main/java/beans/menus/ErrorBean.java

```diff
@@ -0,0 +1,25 @@
+package beans.menus;
```

Comment:
Looks like I accidentally but in the "menus" directory instead of its parent ("beans") where I'm putting all the response-formatting related stuff.